packages/shared/src/components/cards/ActionsButtons/ActionButtons.tsx

```diff
@@ -110,15 +92,20 @@ const ActionButtons = ({
               secondary={isUpvoteActive}
               size={IconSize.Small}
             />
-            {post.numUpvotes ? (
+            {keepUpvoteSpace ? (
               <InteractionCounter
-                className="ml-1.5 tabular-nums"
+                className={classNames(
+                  'ml-1.5 tabular-nums',
+                  !post.numUpvotes && feedActionSpacing && 'invisible',
+                )}
                 value={post.numUpvotes}
               />
             ) : null}
           </Button>
         </SimpleTooltip>
-        <div className="box-border border border-surface-float py-2.5" />
+        {!feedActionSpacing && (
```

Comment:
In your screenshots only 1 version doesn't have the border is that correct?

Reply:
![Screenshot 2024-12-19 at 8 55 05 PM](https://github.com/user-attachments/assets/8be2fc43-5803-437c-ae43-5f5f1a63b049)
![Screenshot 2024-12-19 at 8 55 03 PM](https://github.com/user-attachments/assets/ac00a763-8337-4bcc-8558-fd3218b8f001)
![Screenshot 2024-12-19 at 8 54 57 PM](https://github.com/user-attachments/assets/7f9b0e73-0868-4a0f-a02d-d08087ff5c36)
![Screenshot 2024-12-19 at 8 54 53 PM](https://github.com/user-attachments/assets/9a25e4c2-bf3e-40d3-9011-43f924bb5c0d)
![Screenshot 2024-12-19 at 8 54 51 PM](https://github.com/user-attachments/assets/ed595bb9-d280-49db-b9ff-234fbd9ea228)
![Screenshot 2024-12-19 at 8 54 44 PM](https://github.com/user-attachments/assets/a0eeb2f2-8aca-4e11-afb1-55dfdfba9fee)
![Screenshot 2024-12-19 at 8 54 42 PM](https://github.com/user-attachments/assets/4f5c7c81-de75-47a9-a9b1-b506e312e408)